src/protocols/rdp/channels/rail.c

```diff
@@ -81,11 +69,12 @@ static UINT guac_rdp_rail_complete_handshake(RailClientContext* rail) {
         /* Build number 7600 (0x1DB0) apparently represents Windows 7 and
          * compatibility with RDP 7.0. As of this writing, this is the same
          * build number sent for RAIL connections by xfreerdp. */
-        .buildNumber = 7600
+        .buildNumber = 9600
```

Comment:
This comment is now out of date.

Reply:
Reverted the build number to `7600` - looks like even the current xfreerdp code still uses 7600 (and sometimes 7601), so we'll just stick with that. I do not believe it impacts functionality at all and isn't really required for this change.